CHANGELOG.md

```diff
@@ -7,10 +7,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 
 ### Added
+- **CUMULUS-851** Added workflow lambda versioning feature to allow in-flight workflows to use lambda versions that were in place when a workflow was initiated
+    - Updated Kes custom code to remove logic that used the CMA file key to determine template compilation logic.  Instead, utilize a `customCompilation` template configuration flag to indicate a template should use Cumulus's kes customized methods instead of 'core'.
```

Comment:
I may be misunderstanding but I think this configuration parameter may be unnecessary - we already override `kes` methods such as `compileCF` with the functions in `packages/deployment/app/kes.js`

Reply:
That override is currently being switched via the filename for the CMA being present  in kes.js: 

`if (!filename) return super.compileCF();`

This PR Is updating the aformationed override, and with the additional logic, it no longer makes sense for all templates that are being compiled by the override to each need the CMA filename set.     As such it seemed like a more generic configuration flag was the right thing to do.